mlflow/server/handlers.py

```diff
@@ -669,6 +671,8 @@ def _create_run():
     )
 
     tags = [RunTag(tag.key, tag.value) for tag in request_message.tags]
+    if MLFLOW_RUN_NAME not in [tag.key for tag in tags]:
+        tags.append(RunTag(MLFLOW_RUN_NAME, _generate_random_name()))
```

Comment:
Can we document MLflow automatically generates a random run name somewhere?

Reply:
Just for knowledge purpose. I know that we use all the diff types of stores where we need to set the run_name. Why are we setting it in the `server/handler` as well?

Reply:
yep! Added additional information in docs and added a note in pydoc for `fluent.py`